### DISCUSSION TITLE
Auto-login + configurable return URL following self-activation via email link

### DISCUSSION THREAD

**Discussion by pyrobob83** [ExpressionEngine/ExpressionEngine#3581] [Feature Requests]
### Suggest an idea for ExpressionEngine

Right now, when using member registrations with self-activation via email, the user gets a standard system message and a) they have to login manually after that and b) there's no way to redirect them to a custom template.

Can we please have an option to log users in automatically once they activate via the email link and also have a way to redirect them to a custom template?

@intoeetive Tagging you again because Robin said you were doing a big members update and we agreed you may want to make this part of that.

**Comment by intoeetive** [comment]
I like those ideas! Aiming to implement in https://github.com/ExpressionEngine/ExpressionEngine/pull/3934
